tests/mhd/test_spec.py

```diff
@@ -190,6 +331,10 @@ def test_integrated_stellopt_scenarios_1dof_Garabedian(self):
             surf = SurfaceGarabedian.from_RZFourier(equil.boundary)
             equil.boundary = surf
 
+            # Need to set axis to zero
+            equil.axis['rac'][:] = 0
+            equil.axis['zas'][:] = 0
```

Comment:
It seems unsatisfying that this needs to be done manually. Can this be set automatically as a default?